docs/guide/app.md

```diff
@@ -147,7 +148,39 @@ The addition of `[str]` tells mypy that `run()` is expected to return a string.
 
     Type annotations are entirely optional (but recommended) with Textual.
 
-## CSS
+## Customising your app
```

Comment:
Not sure this hierarchy is required. Can we put the following section under "App Class" after the run method.

Reply:
Come to think of it. If you are mentioning compose, it likely should come later.

I think the "Title and Subtitle" should come after CSS. On a top-level heading of its own.

Just noticed another unrelated issue. "Exiting" is under Widgets. But that feels wrong. Could you promote Exiting to a top header.

Reply:
Addressed in 476eca7.